app/lib/services.rb

```diff
@@ -17,12 +17,12 @@ def self.content_store
   def self.cached_search(params, metric_key: "search.request_time")
     Rails.cache.fetch(params, expires_in: 5.minutes) do
       GovukStatsd.time(metric_key) do
-        rummager.search(params).to_h
+        search_api.search(params).to_h
       end
     end
   end
 
-  def self.rummager
-    @rummager ||= GdsApi::Search.new(Plek.new.find("search"))
+  def self.search_api
```

Comment:
I don't think this method is needed. I think you can call `GdsApi.search` instead of `Services.search_api`: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api.rb#L159

Reply:
Ah that's cool thank you. I will make that change in a separate PR I think as there are quite a lot of tests that will need new stubs if I make the change here.